_projects/SP19-Naruto simulator.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: project
+title: Naruto simulator
+desc: Our game is a first person fighting game that allows user to defeat a strong boss with three unqiue and interesting skills we've designed, all threse skills came from the anime Naruto. The game also include great sound effect and music.
+dl: https://drive.google.com/file/d/1E97SA1jnUzc0FwUvzG0V75KZyFrlAVoC/view?usp=sharing
+semester: Spring 2019
+team: 15
+youtube: 4y4WobB-b90
```

Comment:
Can you make this video accessible through other websites? That way it can play on our embedded video player. Should be an option through YouTube.

Reply:
Just did that, could you check if it works now